init.sql

```diff
@@ -0,0 +1,31 @@
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
```

Comment:
What this line is doing?

Reply:
It a extension for generate uuid v4, Postgres support uuid natively but if we want auto generate thats we use this extension